faf-java-server-app/src/main/java/com/faforever/server/ice/StandardIceServerProvider.java

```diff
@@ -0,0 +1,80 @@
+package com.faforever.server.ice;
+
+import com.faforever.server.config.ServerProperties;
+import com.faforever.server.config.ServerProperties.Ice;
+import com.faforever.server.entity.Player;
+import com.faforever.server.ice.StandardIceServerProvider.StandardIceServerProviderCondition;
+import com.google.common.base.Strings;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.core.env.Environment;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+import org.springframework.stereotype.Service;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import java.net.URI;
+import java.nio.charset.Charset;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.time.Instant;
+import java.util.Base64;
+import java.util.stream.Collectors;
+
+@Service
+@Slf4j
+@Conditional(StandardIceServerProviderCondition.class)
+public class StandardIceServerProvider implements IceServersProvider {
```

Comment:
What about LongTormCrendential(IceServer)Provider?

Reply:
What is that????

Reply:
A better name than standard ice server provider ;)